lib/tools/adb-commands.js

```diff
@@ -728,4 +728,19 @@ methods.getScreenSize = async function() {
   return null;
 };
 
+methods.setHttpProxy = async function (proxyHost, proxyPort) {
```

Comment:
I'd rather rename the variable to proxyUrl or proxyAddress to reflect the fact that it expects not just a simple host name